src/data/density-constants.ts

```diff
@@ -2385,6 +2385,471 @@ export const DIPPRDensityDictionary: Record<string, DIPPRDensityProps> = {
+    minimumTemperature: 172.22,
+    maximumTemperature: 573.5,
+    densityAtMinimumTemperature: 7.0934,
+    densityAtMaximumTemperature: 2.1963,
+  },
+
+  '1,3,5Trinitrobenzene': {
+    name: '1,3,5-Trinitrobenzene',
+    formula: 'C6H3N3O6',
+    CAS: '99-35-4',
+    molecularWeight: 213.10452,
+    C1: 0.48195,
+    C2: 0.23093,
+    C3: 846,
+    C4: 0.28571,
+    minimumTemperature: 398.4,
+    maximumTemperature: 846,
+    densityAtMinimumTemperature: 7.0825,
+    densityAtMaximumTemperature: 2.087,
+  },
+
+  '2,4,6Trinitrotoluene': {
+    name: '2,4,6-Trinitrotoluene',
+    formula: 'C7H5N3O6',
+    CAS: '118-96-7',
+    molecularWeight: 227.1311,
+    C1: 0.37378,
+    C2: 0.21379,
+    C3: 828,
+    C4: 0.29905,
+    minimumTemperature: 354,
+    maximumTemperature: 828,
+    densityAtMinimumTemperature: 6.4521,
+    densityAtMaximumTemperature: 1.7484,
+  },
+
+  Undecane: {
+    name: 'Undecane',
+    formula: 'C11H24',
+    CAS: '1120-21-4',
+    molecularWeight: 156.30826,
+    C1: 0.36703,
+    C2: 0.24876,
+    C3: 639,
+    C4: 0.28571,
+    minimumTemperature: 247.57,
+    maximumTemperature: 639,
+    densityAtMinimumTemperature: 4.9453,
+    densityAtMaximumTemperature: 1.4754,
+  },
+
+  '1Undecanol': {
+    name: '1-Undecanol',
+    formula: 'C11H24O',
+    CAS: '112-42-5',
+    molecularWeight: 172.30766,
+    C1: 0.33113,
+    C2: 0.23676,
+    C3: 703.9,
+    C4: 0.2762,
+    minimumTemperature: 288.45,
+    maximumTemperature: 703.9,
+    densityAtMinimumTemperature: 4.8594,
+    densityAtMaximumTemperature: 1.3986,
+  },
+
+  VinylAcetate: {
+    name: 'Vinyl acetate',
+    formula: 'C4H6O2',
+    CAS: '108-05-4',
+    molecularWeight: 86.08924,
+    C1: 0.9591,
+    C2: 0.2593,
+    C3: 519.13,
+    C4: 0.27448,
+    minimumTemperature: 180.35,
+    maximumTemperature: 519.13,
+    densityAtMinimumTemperature: 12.287,
+    densityAtMaximumTemperature: 3.6988,
+  },
+
+  VinylAcetylene: {
+    name: 'Vinyl acetylene',
+    formula: 'C4H4',
+    CAS: '689-97-4',
+    molecularWeight: 52.07456,
+    C1: 1.2703,
+    C2: 0.26041,
+    C3: 454,
+    C4: 0.297,
+    minimumTemperature: 173.15,
+    maximumTemperature: 454,
+    densityAtMinimumTemperature: 15.664,
+    densityAtMaximumTemperature: 4.8781,
+  },
+
+  VinylChloride: {
+    name: 'Vinyl chloride',
+    formula: 'C2H3Cl',
+    CAS: '75-01-4',
+    molecularWeight: 62.49822,
+    C1: 1.5115,
+    C2: 0.2707,
+    C3: 432,
+    C4: 0.2716,
+    minimumTemperature: 119.36,
+    maximumTemperature: 432,
+    densityAtMinimumTemperature: 18.481,
+    densityAtMaximumTemperature: 5.5837,
+  },
+
+  VinylTrichlorosilane: {
+    name: 'Vinyl trichlorosilane',
+    formula: 'C2H3Cl3Si',
+    CAS: '75-94-5',
+    molecularWeight: 161.48972,
+    C1: 0.59595,
+    C2: 0.24314,
+    C3: 543.15,
+    C4: 0.24856,
+    minimumTemperature: 178.35,
+    maximumTemperature: 543.15,
+    densityAtMinimumTemperature: 8.8236,
+    densityAtMaximumTemperature: 2.4511,
+  },
+
```

Comment:
Water is missing. Ask @maneetgoyal sir, Before adding

Reply:
@sumitnegi7  water and o Terphenyl was added in the beginning of this file , so they need not be added anymore you can checkout it in the last of the file.